src/styles/main.css

```diff
@@ -254,7 +254,7 @@
 
 .form-error {
   /* validation message for the field */
-  @apply text-red text-caption uppercase tracking-wide block mt-1;
+  @apply text-red text-caption uppercase tracking-wide block;
```

Comment:
Why did you decide to remove `mt-1`? Without this class error messages are too close to inputs:
![image](https://user-images.githubusercontent.com/140053/88269458-bc2a9480-ccdc-11ea-80cb-045000bcc3c5.png)

If you are trying to decrease the margin between the Description textarea and the error message, then just add `block` to the textarea and it will be fine:
![image](https://user-images.githubusercontent.com/140053/88269599-feec6c80-ccdc-11ea-9cb9-7174c63e7bbe.png)

In general, we shouldn't solve local styling issues (a field with an error on certain page) changing global styles (a class for all form errors).

Reply:
+1 in this general advice. If you keep changing global stuff you must be ready to review and make sure it applies to all the consumers of that class.

Reply:
My mistake. I'm sorry.  Won't happen again.